next-frontend/build-translations.js

```diff
@@ -19,6 +20,26 @@ function flattenObject(obj, prefix = "") {
   }, {});
 }
 
+// Add Country Names if not translated already
+function addCountryNames(translation, lang) {
```

Comment:
This method could benefit from a little bit of documentation inside its method body. I am mainly confused about all the different variations of `code` appearing here and there